addons/addon-base-raas/packages/base-raas-services/lib/alb/__tests__/alb-service.test.js

```diff
@@ -430,4 +430,63 @@ describe('ALBService', () => {
       }
     });
   });
+
+  describe('modifyRule', () => {
```

Comment:
We should verify if we are calling albClient.modifyRule with correct parameters

Reply:
Added and verified the params of the albClient.modifyRule. Please review